ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/scripts/create_loaded_upgrade_jobs.sh

```
@@ -0,0 +1,102 @@
+    -e "s/version: \"${PRIOR_VERSION}\"/version: \"${TARGET_VERSION}\"/g" \
+    -e "s/variant: aws-${PRIOR_VERSION}-nightly-x86-loaded-upgrade-from-${PRIOR_PRIOR_VERSION}/variant: aws-${TARGET_VERSION}-nightly-x86-loaded-upgrade-from-${PRIOR_VERSION}/g" \
+    "$SOURCE_FILE" > "$TARGET_FILE"
+
+echo "Created: $TARGET_FILE"
+echo ""
+
+# Validate YAML syntax
+echo "Validating YAML syntax..."
+if command -v python3 &> /dev/null; then
+    python3 -c "import yaml; yaml.safe_load(open('$TARGET_FILE'))" 2>&1
+    if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)